weblate/addons/forms.py

```diff
@@ -211,6 +211,10 @@ def __init__(self, *args, **kwargs) -> None:
 
 class JSONCustomizeForm(BaseAddonForm):
     sort_keys = forms.BooleanField(label=gettext_lazy("Sort JSON keys"), required=False)
+    use_compact_separators = forms.BooleanField(
+        label=gettext_lazy("Compact Mode"),
```

Comment:
I think it is better to be more verbose here:

```suggestion
        label=gettext_lazy("Avoid spaces after separators"),
```